drivers/serial/uart_rpi_pico.c

```diff
@@ -302,6 +311,9 @@ static const struct uart_driver_api uart_rpi_driver_api = {
 		.uart_dev = uart##idx,						\
 		.uart_regs = (uart_hw_t *)uart##idx,				\
 		.pcfg = PINCTRL_DT_INST_DEV_CONFIG_GET(idx),			\
+		.reset_ctrl = DEVICE_DT_GET(DT_INST_PHANDLE(idx, resets)),	\
+		.reset_offset = DT_INST_PHA_BY_IDX(idx, resets, 0, offset),	\
+		.reset_bit = DT_INST_PHA_BY_IDX(idx, resets, 0, regbit),	\
```

Comment:
Are these being used?

Reply:
The reset controller is not used right now, it will be used at the moment of a working implementation of a clock controller for pico to replace uart_init() pico sdk function with an implementation independent of their sdk.